doc/classes/Rect2.xml

```diff
@@ -50,63 +50,83 @@
 			<param index="2" name="width" type="float" />
 			<param index="3" name="height" type="float" />
 			<description>
-				Constructs a [Rect2] by x, y, width, and height.
+				Constructs a [Rect2] by setting its [member position] to [param x] and [param y], and its [member size] to [param width] and [param height].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="abs" qualifiers="const">
 			<return type="Rect2" />
 			<description>
-				Returns a [Rect2] with equivalent position and area, modified so that the top-left corner is the origin and [code]width[/code] and [code]height[/code] are positive.
+				Returns a [Rect2] equivalent to this rectangle, with its width and height modified to be non-negative values, and its [member position] turned into the top-left corner of the rectangle.
```

Comment:
Not a native English speaker but I think "turned into" doesn't fit here. Maybe:
```suggestion
				Returns a [Rect2] equivalent to this rectangle, with its width and height modified to be non-negative values, and with its [member position] being the top-left corner of the rectangle.
```
?

Reply:
"To turn into" in this context is a synonym of "to transform" which is pretty much what the method does. However, having non-native speakers asking this is **not** the goal, so this definitely needs to be adjusted.

Reply:
To me "turn into" sounds like "change into completely different thing". But again, maybe it's just me / my ignorance. :upside_down_face: